build.sh

```diff
@@ -15,19 +15,29 @@ echo "Installing dotnet CLI"
 mkdir -p cli
 curl -o cli/dotnet-install.sh -L https://dot.net/v1/dotnet-install.sh
 
+if [ $? -ne 0 ]; then
+	echo "Could not download 'dotnet-install.sh' script. Please check your network and try again!"
+	exit 1
+fi
+
 # Run install.sh for cli
 chmod +x cli/dotnet-install.sh
 
 # v1 needed for some test and bootstrapping testing version
 cli/dotnet-install.sh -i cli -c 1.0
 
+if [ $? -ne 0 ]; then
+	echo ".NET CLI Install failed!!"
+	exit 1
+fi
+
 DOTNET="$(pwd)/cli/dotnet"
 
-echo "$DOTNET msbuild build/config.props /v:m /nologo /t:GetCliBranchForTesting"
+echo "$DOTNET msbuild build/config.props /noAutoRsp /v:m /nologo /t:GetCliBranchForTesting"
```

Comment:
do you have a rsp file in your workspace? 
I imagine the same thing would happen with directory.build.props/targets too.

Reply:
I thought the same thing. Even if there is a global rsp file, it would apply to the all the MSBuild calls except the above ones which may lead to inconsistent local builds.